readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
> You might have mixins for things like media queries that return a whole block of CSS.

I think I wasn't handling this case at all, thats why that thing is called a "Function value". It is literally a CSS value.

Lets discuss more complex things like mixins with media queries now. Can you give me specific examples and evtl. suggestions?